packages/cozy-sharing/src/components/ShareModal/SharingDetailsModal.jsx

```diff
@@ -40,12 +40,14 @@ export const SharingDetailsModal = ({
               isMobile ? 'u-ph-1 u-pt-1-half' : 'u-ph-2 u-pt-1'
             )}
           >
-            <OwnerIdentity
-              name={t(`${documentType}.share.sharedBy`, {
-                name: getDisplayName(owner)
-              })}
-              url={owner.instance}
-            />
+            {owner && (
```

Comment:
I think this will render undefined in the dom.

Let's do: owner ? Component : null